zenodo/modules/deposit/tasks.py

```diff
@@ -120,9 +120,14 @@ def cleanup_indexed_deposits():
          .query('term', **{'_deposit.status': 'draft'})
          .source(['_deposit.id']))
     res = q.scan()
-    es_depids_info = [(d.to_dict().get('_deposit.id', [None])[0], d.meta.id,
-                       d.meta.index, d.meta.doc_type) for d in res]
-    es_depids = {p[0] for p in es_depids_info}
+    erroneous_depids = es_depids_info = []
```

Comment:
```suggestion
    failed_depids, es_depids_info = [], []
```
The word choice is just a nitpick here...

These should be split into two separate statements otherwise you get the following:

```python
In [10]: l1 = l2 = []

In [11]: l1.append('foobar')

In [12]: l1
Out[12]: ['foobar']

In [13]: l2
Out[13]: ['foobar']
```

Reply:
Missed that.